general_good_practice.md

```diff
@@ -49,6 +49,8 @@ At this point, it is probably prudent to check that the github config has been u
 
 `git config --global --list`
 
+Note that, in the output of the above command `user.email` should be one of the email addresses that is assigned to your github user. If this is not the case, then your signed commits will not be verified. To resolve the problem either update `user.email` to match the email address of your github user account, or add the relevant email address from `user.email`to your github user account.
+
```

Comment:
I think it's the email of the GPG key that needs to match one of your GH account emails, it shouldn't be possible to push when `user.email` doesn't match one of your account emails